### DISCUSSION TITLE
When is the new stable version of wgpu going to be released?

### DISCUSSION THREAD

**Discussion by GeezerLMAO** [gfx-rs/wgpu#2648] [Q&A] [answered]
There is a feature that is added that I need (texture arrays, and wgsl syntax improvement) so I want to know when it's going to be released, or is there a way of using the git version of wgpu?

P.S: Sorry for my bad english

**Comment by sqaxomonophonen** [comment, marked as answer]
[Specifying dependencies from git repositories](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

**Comment by CatCode79** [comment]
Specifically you need the `rev` key, which is briefly described in the link already suggested.
A more focused answer:
https://stackoverflow.com/questions/54196846/how-to-specify-a-certain-commit-in-dependencies-in-cargo-toml
